zulip/integrations/openshift/zulip_openshift_config.py

```diff
@@ -28,6 +28,12 @@ def deployment_notice_destination(branch: str) -> Optional[Dict[str, str]]:
     return None
 
 
+# To utilize Zulip's global times (https://zulip.com/help/global-times),
+# uncomment the below function, and call it from format_deployment_message.
+# def get_global_time(dt_str: str) -> str:
+#     return f"<time:{datetime.strptime(dt_str, '%Y-%m-%d %H:%M:%S %z').isoformat()}>"
```

Comment:
This won't work without `datetime` being imported. I think it's fine to have this function available and just unused by default with a comment.

II suggest you also can also put a copy of the datetime=>HTML helper we added in `zerver.lib` into the main `zulip` Python library so it can be used by integrations easily.